lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -57,6 +57,16 @@ HostManager::~HostManager() { llvm::toString(clearHost()); }
 
 llvm::Error HostManager::addNetwork(Module *M) {
   std::lock_guard<std::mutex> networkLock(networkLock_);
+  auto functions = M->getFunctions();
+  for (auto &F : functions) {
+    std::string name = F->getName();
+    auto it = networks_.find(name);
+    if (it != networks_.end()) {
+      return MAKE_ERR(GlowErr::ErrorCode::RUNTIME_REQUEST_REFUSED,
```

Comment:
`RUNTIME_REQUEST_REFUSED` is for inference requests this is for adding network, just `RUNTIME_ERROR` would be better